mpdf.php

```diff
@@ -1416,229 +1418,229 @@ function _getPageFormat($format)
-					break;
-				}
+				$format = array(1729.13, 2437.80);
+				break;
+			}
 			case 'RA2': {
-					$format = array(1218.90, 1729.13);
-					break;
-				}
+				$format = array(1218.90, 1729.13);
+				break;
+			}
 			case 'RA3': {
-					$format = array(864.57, 1218.90);
-					break;
-				}
+				$format = array(864.57, 1218.90);
+				break;
+			}
 			case 'RA4': {
-					$format = array(609.45, 864.57);
-					break;
-				}
+				$format = array(609.45, 864.57);
+				break;
+			}
 			case 'SRA0': {
-					$format = array(2551.18, 3628.35);
-					break;
-				}
+				$format = array(2551.18, 3628.35);
+				break;
+			}
 			case 'SRA1': {
-					$format = array(1814.17, 2551.18);
-					break;
-				}
+				$format = array(1814.17, 2551.18);
+				break;
+			}
 			case 'SRA2': {
-					$format = array(1275.59, 1814.17);
-					break;
-				}
+				$format = array(1275.59, 1814.17);
+				break;
+			}
 			case 'SRA3': {
-					$format = array(907.09, 1275.59);
-					break;
-				}
+				$format = array(907.09, 1275.59);
+				break;
+			}
 			case 'SRA4': {
-					$format = array(637.80, 907.09);
-					break;
-				}
+				$format = array(637.80, 907.09);
+				break;
+			}
 			case 'LETTER': {
-					$format = array(612.00, 792.00);
-					break;
-				}
+				$format = array(612.00, 792.00);
+				break;
+			}
 			case 'LEGAL': {
-					$format = array(612.00, 1008.00);
-					break;
-				}
+				$format = array(612.00, 1008.00);
+				break;
+			}
 			case 'LEDGER': {
-					$format = array(1224.00, 792.00);
-					break;
-				}
+				$format = array(1224.00, 792.00);
+				break;
+			}
 			case 'TABLOID': {
-					$format = array(792.00, 1224.00);
-					break;
-				}
+				$format = array(792.00, 1224.00);
+				break;
+			}
 			case 'EXECUTIVE': {
-					$format = array(521.86, 756.00);
-					break;
-				}
+				$format = array(521.86, 756.00);
+				break;
+			}
 			case 'FOLIO': {
-					$format = array(612.00, 936.00);
-					break;
-				}
+				$format = array(612.00, 936.00);
+				break;
+			}
 			case 'B': {
-					$format = array(362.83, 561.26);
-					break;
-				}  //	'B' format paperback size 128x198mm
+				$format = array(362.83, 561.26);
+				break;
+			}  //	'B' format paperback size 128x198mm
 			case 'A': {
-					$format = array(314.65, 504.57);
-					break;
-				}  //	'A' format paperback size 111x178mm
+				$format = array(314.65, 504.57);
+				break;
+			}  //	'A' format paperback size 111x178mm
 			case 'DEMY': {
-					$format = array(382.68, 612.28);
-					break;
-				}  //	'Demy' format paperback size 135x216mm
+				$format = array(382.68, 612.28);
+				break;
+			}  //	'Demy' format paperback size 135x216mm
 			case 'ROYAL': {
-					$format = array(433.70, 663.30);
-					break;
-				} //	'Royal' format paperback size 153x234mm
+				$format = array(433.70, 663.30);
+				break;
+			} //	'Royal' format paperback size 153x234mm
 			default: {
-					$format = array(595.28, 841.89);
-					break;
-				}
+				$format = array(595.28, 841.89);
+				break;
+			}
 		}
```

Comment:
These changes have no connection to the theme of the PR, please keep as it were.